CMakeLists.txt

```diff
@@ -120,8 +120,23 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/ebml_export.h DESTINATION ${CMAKE_INST
 if(NOT DISABLE_PKGCONFIG)
   set(prefix ${CMAKE_INSTALL_PREFIX})
   set(exec_prefix "\$\{prefix\}")
-  set(libdir "\$\{prefix\}/${CMAKE_INSTALL_LIBDIR}")
-  set(includedir "\$\{prefix\}/${CMAKE_INSTALL_INCLUDEDIR}")
+  if(${CMAKE_VERSION} VERSION_LESS "3.20.0") 
+    set(libdir "\$\{prefix\}/${CMAKE_INSTALL_LIBDIR}")
+    set(includedir "\$\{prefix\}/${CMAKE_INSTALL_INCLUDEDIR}")
+  else()
+    cmake_path(IS_ABSOLUTE CMAKE_INSTALL_LIBDIR abs_libdir)
+    cmake_path(IS_ABSOLUTE CMAKE_INSTALL_INCLUDEDIR abs_incdir)
+    if ($abs_libdir)
```

Comment:
As far as I understand, this is both unnecessarily contrived and unnecessarily demanding on the CMake version: `cmake_path` is ≥ 3.20 only, but `if (IS_ABSOLUTE ...)` was introduced much earlier.

Reply:
cmake_path[¶](https://cmake.org/cmake/help/v3.20/command/cmake_path.html#cmake-path) doesn't exist before 3.20

Reply:
Ah yes, there's `if (IS_ABSOLUTE )` in [CMake 3.1](https://cmake.org/cmake/help/v3.1/command/if.html).